### ISSUE TITLE
Rename to "boilerplate" rather than "framework"?

### ISSUE THREAD

**Issue by markdalgleish** [jalxob/cool-kitten#7]
Hey, author of Stellar.js here. I really appreciate that you like my plugin enough to include it in your first open source project :)

I just thought I should raise the question about whether this project should be referred to as "boilerplate" as opposed to a "framework".

You can get a good idea of what constitutes a framework here: http://net.tutsplus.com/articles/web-roundups/20-javascript-frameworks-worth-checking-out/

Now, compare that to this list of web dev boilerplate packages: http://mashable.com/2011/06/13/boilerplates-templates-dev/

I think making this change would avoid confusion. I see that you've bundled your own responsive grid, so it might be a better idea to create a new project specifically for your grid styles (with an equally fun name, of course), then include it in the "Cool Kitten" boilerplate package. What do you think? :)

**Comment by FireZenk** [comment]
I think the same, so +1

**Comment by jalxob** [comment]
It sounds great. I will do those changes asap. Thank you so much for your comment and your amazing plugin.

**Comment by jbeja** [comment]
1+

**Comment by madahmani** [comment]
Agree too and very good work on this "Boilerplate" jalxob!

**Comment by jankuca** [comment]
+1 I was confused when I did not find any actual parallax code in the repo other than (minified!) Stellar. I kind of feel like this is a Stellar demo, not a project on itself.

From what I understood, the only thing you are supplying is the CSS code for the grid. How is parallax related to this? I was expecting something along the lines of having multiple grids layered on top of each other moving in different speeds—that would surely constitute as a "responsive parallax framework".

It would be really useful to have some examples other that the grid itself it there is more that this project offers. Thanks!

**Comment by madahmani** [comment]
I disagree with all who think this project is not a good "framework" even if it is very basic, the author made a very clean job.
I lately made a project using it: http://www.sandbarmissionbeach.com.au/ and it was a success in no time!

**Comment by jankuca** [comment]
In that case, can you explain how you used it? Why would Stellar.js on its own not be sufficient enough?

BTW the background is fixed, it does not scroll. Is that the intended behavior? I do not see any parallax elements there.

**Comment by madahmani** [comment]
Please just take some time to study the source of the page, it is very easy to implement.
Yes, my background is fixed, upon client choice but the original example is scrolling both containers.

Please keep in mind that this is very basic compared to stellar or jKit.

As I understood, Jalxob wanted to make a parallax scrolling effect, so what you really have is the scroll along with the parallax.

If you are looking for a more elaborate framework please read here: http://jquery-jkit.com/commands/parallax.html

and here: http://webdesign.tutsplus.com/tutorials/complete-websites/create-a-parallax-scrolling-website-using-stellar-js/

**Comment by madahmani** [comment]
And here is one last gift for you to study: http://pepsized.com/demo/?id=261
